src/gluonts/ext/r_forecast/_hierarchical_predictor.py

```diff
@@ -99,6 +103,7 @@ def __init__(
         period: int = None,
         trunc_length: Optional[int] = None,
         params: Optional[Dict] = None,
+        level: Optional[int] = None,
```

Comment:
Could you add this to the docstring as well?

Reply:
Added `level` as requested.